wgpu-core/src/instance.rs

```diff
@@ -342,18 +344,22 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                 let map = |inst| {
                     inst
                     .as_ref()
-                    .and_then(|inst| inst.create_surface(handle).ok())
+                    .and_then(|inst| inst.create_surface(handle).map_err(|e| {
+                        eprintln!("Error: {:?}", e);
+                    }).ok())
```

Comment:
I ran into an `UnsupportedWindowHandle` error when I accidentally left out the X11 feature, but because the code previously just ran `.ok()` I had no way of knowing what was wrong. I'm not sure if this is the right fix or not, but it definitely helps if your surface creation fails. :wink:

This is in a separate commit that can be trivially moved to another PR or just dopped completely, it is up to you.

Reply:
let's make this `tracing::warn!` instead of `eprintln`.